app/views/api/v1/split_details/show.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.split_name @split_detail.split.name
+json.hypothesis @split_detail.hypothesis
+json.assignment_criteria @split_detail.assignment_criteria
+json.description @split_detail.description
+json.owner @split_detail.owner
+json.location @split_detail.location
+json.platform @split_detail.platform
```

Comment:
there's a shorthand you can use for pulling values and assigning them to keys of the same name:

```
json.(@split_detail, :hypothesis, :assignment_criteria, ...)
```